Dockerfile

```diff
@@ -0,0 +1,106 @@
+# Build:
+#   docker build -t discourse/ruby:3.3-bookworm-slim --build-arg DEBIAN_RELEASE=bookworm --build-arg RUBY_VERSION=3.3.3 .
+#
+# You are not expected to build and push this Docker image by hand. This is done by the `build` Github Actions workflow.
+ARG DEBIAN_RELEASE=
+ARG RUBY_VERSION=
+
+FROM debian:${DEBIAN_RELEASE}-slim
+
+ARG RUBY_VERSION
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+ENV RUBY_VERSION=${RUBY_VERSION}
+
+# Installs system dependencies required to run Ruby
+RUN set -eux; \
```

Comment:
I wonder if it would be better to chain the commands using `&&` instead of `set -eux` and `;`.

Reply:
However, I might leave it be because, the more I look at it, I understand how complex it is.

Reply:
I followed what https://github.com/docker-library/ruby did here. Not sure if either way is better.